component/main.jsonnet

```
@@ -26,7 +26,7 @@ local template = com.namespaced(params.namespace, kube.Secret('oauth-templates')
   },
 });
 
-local idps = params.identityProviders;
+local idps = std.prune(params.identityProviders);
```

Consider using the following instead of `std.prune()`, if you expect the object which you're pruning to be deeply nested (this is primarily a performance question, not sure it's necessary here):

```suggestion
local idps = std.filter(function(it) it != null, params.identityProviders);
```

---

Doesn't work as suggested:
```console
Jsonnet error: failed to compile /tmp/component-t_gxysa4/dependencies/openshift4-authentication/component/main.jsonnet:
 RUNTIME ERROR: Builtin function filter expected (function, array) but got (function, object)
	/tmp/component-t_gxysa4/dependencies/openshift4-authentication/component/main.jsonnet:29:14-75	thunk <idps>
```